app/serializers/spree_multi_vendor/v2/storefront/product_serializer_decorator.rb

```diff
@@ -0,0 +1,13 @@
+module SpreeMultiVendor
```

Comment:
@poudelprakash we don't need this decorator anymore, already done

Reply:
I don't understand where it is already done..
Is it already merged?

Reply:
> I don't understand where it is already done..
> Is it already merged?

https://github.com/spree-contrib/spree_multi_vendor/blob/9619f3a68cb253c53d2ba7b697267e2d93c0ca47/app/serializers/spree_multi_vendor/vendor_models_decorator.rb#L1

Reply:
Awesome!

I have removed the decorator :)